operator/controllers/seldondeployment_explainers.go

```diff
@@ -247,7 +247,7 @@ func createExplainerIstioResources(pSvcName string, p *machinelearningv1.Predict
 						{
 							Uri: &istio_networking.StringMatch{MatchType: &istio_networking.StringMatch_Prefix{Prefix: "/seldon.protos.Seldon/"}},
 							Headers: map[string]*istio_networking.StringMatch{
-								"seldon":    &istio_networking.StringMatch{MatchType: &istio_networking.StringMatch_Exact{Exact: mlDep.Name}},
+								"seldon":    &istio_networking.StringMatch{MatchType: &istio_networking.StringMatch_Exact{Exact: mlDep.GetName()}},
```

Comment:
Interesting, I didn't know that `ObjectMeta` also had methods defined!

Reply:
No, those are methods that we added via composition, it's really one that we added to make the name with a hash if too long. Actually, this may have been intended tho - @cliveseldon could you confirm if this was intended?

Reply:
Which code are you referring to?

Reply:
oh - those are just convenience methods added by k8s. `name` and `GetName` are the same.